presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -333,6 +340,11 @@ public static boolean isBucketExecutionEnabled(ConnectorSession session)
         return session.getProperty(BUCKET_EXECUTION_ENABLED, Boolean.class);
     }
 
+    public static boolean shouldIgnoreTableBucketing(ConnectorSession session)
```

Comment:
Do you feel strong about `should`? Usually we just use `is*`

Reply:
Agree. Why use `should` here ? :)

Reply:
because `should` makes more sense in English.  "is ignore table bucketing" isn't a coherent  phrase, but "should ignore table bucketing" is.